REQUIRE

```diff
@@ -1,5 +1,5 @@
-julia 0.4
-Compat 0.8.5
-Distances 0.3.1
-NearestNeighbors 0.0.3
-StatsBase 0.9.0
```

Comment:
The lower bounds on the dependencies should not be removed—those are important to have there